### ISSUE TITLE
Removing ENS Tabsheet

### ISSUE THREAD

**Issue by johnshearing** [kvhnuke/etherwallet#878]
I am using MEW on an air-gapped raspberry pi. 
I am using v3.8.3.
As per Taylor's instructions I am able to make custom changes. 
Thanks Taylor.
I wanted to remove all tabsheets except for the Off Line Transactions tabsheet.
In globalService.js there is an object named **tabs**, inside of which there are objects for controlling the properties of all available tabsheets.
Setting the **mew** property to **false** removes the tabsheet for all tabsheets except for one of them.
The ENS tabsheet can not be removed by setting the **mew** property to **false**.
I looked as best I could to figure out why.
I was unable to see the reason.
Would you please tell me how to remove this tabsheet?
Also, what technique could I use to figure this out for myself next time?
In other words: Is there something I should have been searching for, or is there some way to watch MEW work in a debugger or is there some other way to figure this out?
Thanks John

**Comment by tayvano** [comment]
Instead of doing it as you are, go to signmsg.html or whatever and replace it with the offline template instead of the sign message. This is going to be far easier and cleaner. Let me kmow if you need more assistance.

**Comment by johnshearing** [comment]
Hi Taylor,
Thanks for the quick response.

I was trying to remove the ENS tabsheet.
So following your advice as best I understood:
I copied the contents of offlineTx.tpl to ens.tpl.
This didn't work so I opened index.html and tried replacing the @@include for ens.tpl with the @@include for offlineTx.tpl. That didn't remove the ENS tabsheet either but it did empty out the contents. 

That's good enough for right now. It is a very small thing. 

MEW works beautifully and I have all the other customizations I need right now. I will come back to the problem in a few months after acquiring a better understanding of how to work with AngularJS and MEW in general.

Thanks for giving regular people a secure way to manage their own private keys.

**Comment by yzs981130** [comment]
same question
changes about ENS in globalService.js doesn't work, changes of any other tabs work.

eg. 
```js
ens: {
    id:7,
    name: "NAV_ENS",
    url: "ens",
    mew: false,
    cx: false
  },
```
can't remove ENS tab in index.html

Thanks a lot.
